### DISCUSSION TITLE
Help me if you can

### DISCUSSION THREAD

**Discussion by pni87** [arendst/Tasmota#13493] [Q&A] [answered]
hi, i'm writing from italy so sorry for my english...
I tasmotized a sonoff basic and I modified to have the contact clean. i need the sonoff for switch on and off the pellet stove
with pulsetime so I simulate pressing the button for 3 seconds. The problem is : my stove goes in screensaver, so when i turn on the sonoff, the stove doesnt switch on but only the screen scwitch on, so i need a double command a pulsetime for like 1 second for exit from the standy, then a second pulsetime for 3 seconds for turn on the stove. So i think i need a rule??? Sorry i really can't do it, so i need your help, i think for you guys is really simple...for me it s like going to the moon ahah. So i need a command on the console of tasmota sonoff that when i push on the realy goes on for one sec then off then another pulsetime of 3 sec, i hope u can understand me and help me thanks

**Comment by ascillato** [comment, marked as answer]
Hi, as you didn't have any answer here, you can try addressing this to the [Tasmota Support Chat](https://discord.gg/Ks2Kzd4).
